### DISCUSSION TITLE
Isotype removal

### DISCUSSION THREAD

**Discussion by signaltwo** [campbio/decontX#26] [General]
Thanks for sharing this exciting tool! I am applying decontPro to a TotalSeq experiment with 155 markers and 9 isotypes. In your paper Isotypes were removed before running decontPro() - is this absolutely essential? Is there any harm to the model when leaving isotypes in? 

Further, do you have any advice for tuning parameters when applying decontPro to a larger feature set (e.g. 155 antibodies) Thank you!

**Comment by joshua-d-campbell** [comment]
Hi @tsbdubensky, thanks for trying out our tool. You can leave the isotypes in. We took them out in the paper to demonstrate that we didn't need them for the background estimation/correction whereas other tools require them.

We have successfully applied it to Total-seq panels of >100 proteins. The tool takes a while to run but should still work with larger panels. 

Let us know if you have other issues.

**Comment by signaltwo** [comment]
Thanks! I am also wondering what the optimal order of ADT-based quality control filtering and running decontPro() would be.

In your paper I see that "Datasets were preprocessed by removing HTO tags and isotypes, filtering out cell droplets with top and bottom one percent of ADT and RNA total library sizes, and droplets with 15% or higher mitochondrial gene counts."

Should more stringent filtering of ADT (# UMI and # Features) only be performed after running decontPro? Does removing many low ADT count cells before running decontPro impair modeling of the background and ambient count distribution?

Or can we perform QC on RNA and ADT as normal, perform an initial ADT-based clustering, run decontPro, and then proceed with downstream analysis using the cleaned ADT counts?

**Comment by joshua-d-campbell** [comment]
This is a very good question. You can start by removing cells with very low ADT counts. I think you can then run the initial clustering and decontPro on the ADT data as normal. If there are small subsets of cells that have extremely high numbers of many ADTs, you may want to filter these out and rerun the tool. I'm going to convert this to a Discussion so others can see it.
